server/src-lib/Hasura/Eventing/ScheduledTrigger.hs

```diff
@@ -192,26 +191,32 @@ data ScheduledEventFull
   , sefRetryConf     :: !STRetryConf
   , sefHeaders       :: ![EventHeaderInfo]
   , sefComment       :: !(Maybe Text)
+  , sefCreatedAt     :: !(Maybe UTCTime)
```

Comment:
This is arguable so I leave it up to you. But it feels the internal representation of `ScheduledEventFull` can have `UTCTime` for both cron and one-off events.

We can avoid exposing it in the payload for cron events because it is not user-facing.

This would mean we change the type to `sefCreatedAt     :: !UTCTime` and add the comment for `sewpCreatedAt`.

Reply:
> But it feels the internal representation of ScheduledEventFull can have UTCTime for both cron and one-off events.

Yes, we can have, but what's the point of having the `sefCreatedAt` for cron events, if it's not going to be used anywhere. 

But, now that I think of it, maybe it'll be useful to include it in the logs.

Reply:
I have included the `created_at` field in `CronEventPartial` as well and not sending the value in the payload.